helm/cluster-openstack/templates/_cluster.tpl

```diff
@@ -1,21 +1,23 @@
+{{- define "cluster" }}
 apiVersion: cluster.x-k8s.io/v1alpha4
 kind: Cluster
 metadata:
   annotations:
-    cluster.giantswarm.io/description: {{ .Values.clusterDescription }}
+    cluster.giantswarm.io/description: "{{ .Values.clusterDescription }}"
   labels:
-    {{- include "labels.common" . | nindent 4 }}
-  name: {{ include "resource.default.name" . }}
+    cluster-apps-operator.giantswarm.io/watching: ""
```

Comment:
Without Kyverno policies to default this, we need to specify it in the template so cluster-apps-operator will install apps.